server.js

```diff
@@ -25,6 +25,19 @@ if (process.env.NODE_ENV !== "test" && (!MATOMO_HOST || !MATOMO_SITE_ID || !MATO
   process.exit(1);
 }
 
+if (process.env.NODE_ENV === "test") {
+  if (!process.env.ECOBALYSE_DATA_DIR) {
```

Comment:
This could be `if (process.env.NODE_ENV === "test" && !process.env.ECOBALYSE_DATA_DIR)`

Even better, if you destructure vars along the ones at the beginning of the file:

```js
// Env vars
const {
  ECOBALYSE_DATA_DIR,
  MATOMO_HOST, 
  MATOMO_SITE_ID, 
  MATOMO_TOKEN, 
  NODE_ENV,
  SENTRY_DSN, 
} = process.env;

// …

if (NODE_ENV === "test" && !ECOBALYSE_DATA_DIR) {
```

(also yeah I like we order things alphabetically everywhere we can ^^)

Reply:
Also, when `ECOBALYSE_DATA_DIR` is defined in env, we should test that the directory and detailed processes data files exist, and raise/exit early if they don't — meaning we don't ever care about that later on.